SKILL.tsv

```diff
@@ -5354,46 +5354,46 @@ SKILL_20151016_005353	$Causes instant kill on small-type monsters by 0.5% chance
 SKILL_20151016_005354	$Zalciai: Enhance
 SKILL_20151016_005355	$Increases the critical attack of [Zalciai] by 1 per attribute level.
 SKILL_20151016_005356	$Lv. 3 Zalciai required
-SKILL_20151022_005357	
+SKILL_20151022_005357	{#DD5500}{ol}[Physical] - [Slash]{/}{/}{nl}Lift your sword up perform a vertical strike downward. Medium and large size enemies suffering defence reduction from [Skull Swing] take additional damage proportional to the reduced amount of defense. The enemy also takes additional damage according to the number of debuffs in effect.
 SKILL_20151022_005358	
 SKILL_20151022_005359	
 SKILL_20151022_005360	
 SKILL_20151022_005361	$Attaque Composee
-SKILL_20151022_005362	
+SKILL_20151022_005362	${#DD5500}{ol}[Physical] - [Pierce]{/}{/}Stab an enemy in front of you.
 SKILL_20151022_005363	$Lunge
-SKILL_20151022_005364	
-SKILL_20151022_005365	
+SKILL_20151022_005364	${#DD5500}{ol}[Physical] - [Pierce]{/}{/}Lower your stance and reach out to attack an enemy. Your evasion rate increases and the enemy becomes more vulnerable to [Slash] attacks.
+SKILL_20151022_005365	$Attack: #{SkillAtkAdd}#{nl}Attribute Damage: +#{CaptionRatio}#%{nl}Evasion Rate: #{CaptionRatio2}#%{nl}Duration: #{CaptionTime}# seconds
```

Comment:
Remove the + sign in front of Attribute Damage. It needs to be in front of Evasion Rate.

-> Evasion Rate: +#{CaptionRatio2}#%